annet/gen.py

```diff
@@ -260,7 +260,7 @@ def _old_new_per_device(ctx: OldNewDeviceContext, device: Device, filterer: Filt
         if ctx.args.acl_safe:
             safe_new_files = res.new_files(safe=True)
             safe_new_json_fragment_files = res.new_json_fragment_files(old_json_fragment_files, safe=True)
-            if filters:
+            if len(filters):
```

Comment:
lets not ignore pep8 for no reason

https://peps.python.org/pep-0008/#programming-recommendations

```
For sequences, (strings, lists, tuples), use the fact that empty sequences are false:
# Correct:
if not seq:
if seq:
```